### PR TITLE
Allow user to specify metalsmith `ignore` option

### LINKED ISSUE #197: Feature: adding an "ignore" option in docpress.json
When using docpress in a large code base it would be nice to tell the watcher to ignore certain directories. Not only can this improve the speed but it also fixes a problem I recently discovered where building the docs will not work because it's trying to reach inside of a electron app build.

### PR DESCRIPTION
**Pull request by kjantzer**
Related to #91; also helps solve docpress/docpress#230
Also fixes #197

### CODE REVIEW

File: lib/ms.js [hunk 1]
```diff
@@ -23,6 +23,9 @@ function docpress (cwd, options) {
     .metadata(meta)
     .ignore('node_modules')
     .ignore(`!**/{${meta.docs}{,/**/*},*.md}`)
+    
+  if( typeof meta.ignore == 'object' && meta.ignore.forEach )
```
Comment (knownasilya) [inline lib/ms.js]: Could you use {} around the body and remove the extra spaces in the ()? Just like the other code in this file.

Comment (knownasilya) [reply lib/ms.js]: Also the space after `if`.

Comment (kjantzer) [reply lib/ms.js]: Done

File: lib/ms.js [hunk 2]
```diff
@@ -23,6 +23,10 @@ function docpress (cwd, options) {
     .metadata(meta)
     .ignore('node_modules')
     .ignore(`!**/{${meta.docs}{,/**/*},*.md}`)
+    
+  if (typeof meta.ignore == 'object' && meta.ignore.forEach) {
```
Comment (knownasilya) [inline lib/ms.js]: I think we can use `Array.isArray(meta.ignore)` and then use `forEach`.

Comment (kjantzer) [reply lib/ms.js]: Good point. Didn't realize `Array.isArray` existed. Learn something new everyday!

**Comment by knownasilya** [comment]
Thanks!

**Pull request merged**